javascript_odev/hafta#2/16-ramazan/calc.js

```diff
@@ -0,0 +1,52 @@
+function calc( sayi_1, sayi_2, islem ) {
+    var result = {
+     sonuc : 0,
+     hata : ""
+    }
+    
+    if (typeof sayi_1 !== 'number' || typeof sayi_2 !== 'number') {
+        result.hata = 'Lütfen sayi giriniz.'
+         }
+
+else if ( islem == "/" && sayi_2 == 0) {
```

Comment:
elinize sağlık, 
else if  yapıları fonksiyonun içerisinde, bundan dolayı bir tab kadar içerde yazılması kodun okunalabilirliği için iyi olacaktır